### ISSUE TITLE
loss oscillate 

### ISSUE THREAD

**Issue by ilovecv** [baaivision/MUSE-Pytorch#3]
Hi,

Thanks for sharing the awesome work!

I am training the model on cc3m. However, after training for 50K steps, I found the loss started to oscillate. Did you observe similar phenomenon during training? I tried to reduce the learning rate by 10 times, but it didn't help. Do you have any suggestions?

Thanks!

![Screenshot 2023-06-14 at 3 51 21 PM](https://github.com/baaivision/MUSE-Pytorch/assets/11083384/50af7ce6-e3fd-4a4d-9f04-13ec15d35b07)

**Comment by ogkalu2** [comment]
@ilovecv 

Hey. Did this change ?

Also May I ask how much it cost, how long it took and what systems you used to train ?

**Comment by liqiangniu** [comment]
How many gpus are you used?  A100 or V100?

**Comment by liqiangniu** [comment]
@ilovecv
